src/python/library/tritonclient/grpc/aio/__init__.py

```diff
@@ -636,7 +636,7 @@ async def stream_infer(
 
         Parameters
         ----------
-        inputs_iterator : async_generator
+        inputs_iterator : asynchronous iterator
```

Comment:
This is always "asynchronous iterator" not "asynchronous generator function", because `inputs_iterator` is used directly without `inputs_iterator()`.